ansible/roles/ocp4-workload-ccnrd/files/knative-sources.yaml

```diff
@@ -801,7 +804,20 @@ data:
     #   PodIP         string  // IP of the pod hosting the revision
     # }
     #
-    logging.request-log-template: '{"httpRequest": {"requestMethod": "{{.Request.Method}}", "requestUrl": "{{js .Request.RequestURI}}", "requestSize": "{{.Request.ContentLength}}", "status": {{.Response.Code}}, "responseSize": "{{.Response.Size}}", "userAgent": "{{js .Request.UserAgent}}", "remoteIp": "{{js .Request.RemoteAddr}}", "serverIp": "{{.Revision.PodIP}}", "referer": "{{js .Request.Referer}}", "latency": "{{.Response.Latency}}s", "protocol": "{{.Request.Proto}}"}, "traceId": "{{index .Request.Header "X-B3-Traceid"}}"}'
+    logging.request-log-template: >-
```

Comment:
yaml to json conversion includes the newlines here?
`{
   "logging.request-log-template": "{\"httpRequest\":  {\"requestMethod\": \"{{.Request.Method}}\",\n   \"requestUrl\": \"{{js .Request.RequestURI}}\",\n   \"requestSize\": \"{{.Request.ContentLength}}\",\n   \"status\": \"{{.Response.Code}}\",\n   \"responseSize\": \"{{.Response.Size}}\",\n   \"userAgent\": \"{{js .Request.UserAgent}}\",\n   \"remoteIp\": \"{{js .Request.RemoteAddr}}\",\n   \"serverIp\": \"{{.Revision.PodIP}}\",\n   \"referer\": \"{{js .Request.Referer}}\",\n   \"latency\": \"{{.Response.Latency}}s\",\n   \"protocol\": \"{{.Request.Proto}}\"},\n   \"traceId\": \"{{index .Request.Header \"X-B3-Traceid\"}}\"}"
}` is the output I get from yaml to json.

Reply:
Odd, that surprises me too. I am not sure of a readable way of formatting this to make it readable... but we need to find one or add an exception.

Reply:
Will add an exception for now